test/e2e/eventcounter/eventcounter_test.go

```diff
@@ -53,8 +53,8 @@ import (
 
 const (
 	prefixTestName = "e2e-test"
-	timeout        = 20 * time.Second
-	retryTimes     = 30
+	timeout        = 60 * time.Second
```

Comment:
During the tests, I saw a few times a context deadline was exceeded while inserting data in BigQuery. So, I changed it to 60 seconds as the dev environment is configured.